repository/scripts/area/wilderness.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2017 Virtue Studios
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+/* globals EventType */
+var coords = require('../map/coords');
+var chat = require('../chat');
+var map = require('../map');
+var anim = require('../core/anim');
+
+module.exports = (function () {
+	return {
+		init : init
+	};
+	
+	function init (scriptManager) {
+		scriptManager.bind(EventType.OPLOC1, [65084,65086,65082,65076,65077,65079], function (ctx) {//Wildy Ditch
+		if(map.getCoordY(ctx.player) == 3520) {
```

Comment:
Not a huge issue, but could you check your formatting settings in your text editor? We should have one level of indentation per control flow block to make the code readable, for example:
```
if (a === 2) {
    var b = 0;
    while (b != 3) {
        b++;
    }
}
```
This just makes the code more readable than having everything at the same level:
```
if (a === 2) {
var b = 0;
while (b != 3) {
b++;
}
}
```